# Automatically Generated in 2017-06-12 22:33:47.452827
# MESSAGE: CONTROL_COMMAND
# Description:Control command data
uint8 ID = 64
uint8 sysid
uint8 compid
float32 velocity
float32 steering_front
float32 steering_rear
uint8 indicator_left
uint8 indicator_right
